oak/common/app_config_serializer.cc

```diff
@@ -14,46 +14,71 @@
  * limitations under the License.
  */
 
+#include <map>
 #include <memory>
 #include <string>
 
 #include "absl/flags/flag.h"
 #include "absl/flags/parse.h"
+#include "absl/memory/memory.h"
+#include "absl/strings/str_split.h"
 #include "asylo/util/logging.h"
+#include "google/protobuf/text_format.h"
 #include "oak/common/app_config.h"
 #include "oak/common/utils.h"
 
-ABSL_FLAG(std::string, module, "", "File containing the compiled WebAssembly module");
-ABSL_FLAG(bool, logging, false, "Enable application logging");
-ABSL_FLAG(std::string, storage_address, "127.0.0.1:7867",
-          "Address ot the storage provider to connect to");
-ABSL_FLAG(int, grpc_port, 8080, "Port for the Application to listen on");
+ABSL_FLAG(std::string, textproto, "", "Textproto file with application configuration");
+ABSL_FLAG(std::vector<std::string>, modules, std::vector<std::string>{},
+          "A comma-separated list of entries `module=path` "
```

Comment:
'module' is supposed to match the name of a `NodeConfiguration`, yes?  It would be nice to find a better label but I can't immediately think of one.

Reply:
Yes.
We need to somehow identify them, I thought it was the easiest option.